src/editors/data/redux/thunkActions/video.js

```diff
@@ -88,25 +97,34 @@ export const determineVideoSource = ({
   };
 };
 
-// copied from frontend-app-learning/src/courseware/course/course-license/CourseLicense.jsx
+// partially copied from frontend-app-learning/src/courseware/course/course-license/CourseLicense.jsx
 // in the long run, should be shared (perhaps one day the learning MFE will depend on this repo)
```

Comment:
I'm not sure I agree with this long term vision entirely `one day the learning MFE will depend on this repo`. I think that this lib should move into the authoring MFE.

Reply:
Note: this is the place for that discussion though